portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,54 @@
 <html>
```

Comment:
As a general rule, try running it through a validator, to catch anything that isn't easy for a human to catch.